src/libraries/System.Private.CoreLib/src/System/ByReference.cs

```diff
@@ -17,7 +17,7 @@ namespace System
 #pragma warning restore CA1823, 169
 
         [Intrinsic]
-        public ByReference(ref T value)
+        public ByReference(ref T _)
```

Comment:
Similar here. The argument is actually used by the JIT intrinsic. It is misleading to tag is as unused.

Reply:
moved to pragma